indra/newview/llpanelface.cpp

```diff
@@ -5188,6 +5208,17 @@ void LLPanelFace::onTextureSelectionChanged(LLInventoryItem* itemp)
 
 void LLPanelFace::onPbrSelectionChanged(LLInventoryItem* itemp)
 {
+    // Commit transform values before switching PBR materials
+    if (mPBRScaleU && mPBRScaleV && mPBROffsetU && mPBROffsetV && mPBRRotate)
+    {
+        onCommitGLTFTextureScaleU();
+        onCommitGLTFTextureScaleV();
+        onCommitGLTFTextureOffsetU();
+        onCommitGLTFTextureOffsetV();
+        onCommitGLTFRotation();
```

Comment:
This causes 5 individual updates, 5 different overrides will be racing each other to the server.